kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/koin/KordExContext.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.kotlindiscord.kord.extensions.koin
+
+import org.koin.core.Koin
+import org.koin.core.KoinApplication
+import org.koin.core.context.KoinContext
+import org.koin.core.error.KoinAppAlreadyStartedException
+import org.koin.core.module.Module
+import org.koin.dsl.KoinAppDeclaration
+
+/**
+ * The [KoinContext] for bot instances.
+ *
+ * This contains the [KoinApplication] and its [Koin] instance for dependency injection.
+ *
+ * To use this context, implement [KordExKoinComponent].
+ */
+public object KordExContext : KoinContext {
+    /** The current [Koin] instance. */
+    private var koin: Koin? = null
+
+    /** The current [KoinApplication]. */
+    private var koinApp: KoinApplication? = null
+
+    /** Gets the [Koin] instance.
```

Comment:
Put that on the next line, please, as is the convention in this project. You can assume this for the rest of the multi-line block comments in this file as well.